bacula-web/bacula-web/CVE-2017-15367.yaml

```diff
@@ -0,0 +1,8 @@
+title: "Bacula-web before 8.0.0-rc2 is affected by multiple SQL Injection vulnerabilities"
+link: https://github.com/bacula-web/bacula-web/releases/tag/v8.0.0-rc.2
+cve: CVE-2017-15367
+branches:
+    master:
+        time:     2018-03-02 16:41:32
+        versions: ['>8.0.0', '<8.0.0']
```

Comment:
this won't match any version when applying the range

Reply:
Thanks for your feedback.

I tried to define something like ['>8.0.0-rc.1', '<8.0.0-rc.2'] or ['<8.0.0-rc.2'].
But trying to validate my commit I received with the error message
`Version constraint "is not in an acceptable format"`